sdks/python/apache_beam/io/avroio.py

```diff
@@ -90,7 +78,7 @@ def __init__(
       file_pattern=None,
       min_bundle_size=0,
       validate=True,
-      use_fastavro=_use_fastavro()):
+      use_fastavro=True):
```

Comment:
So, to clarify, does both Avro and fastavro work for Python even though we set fastavro to default ?

Reply:
It's possible to force FastAvro -> Avro, although we currently have avro==1.8.2 on Dataflow workers, which is outdated and may have Py3 bugs. It's unclear if it's safe to update.
There is a comment: 

    # Do not upgrade avro library to 1.9.0 until
    # https://issues.apache.org/jira/browse/AVRO-2429 has been
    # resolved (see http://b/133317612 for more context)

Reply:
This was fixed in Avro 1.9.2: https://issues.apache.org/jira/browse/AVRO-2429
So it should be safe to upgrade to that version.